tests/queries/loader.go

```diff
@@ -16,19 +16,19 @@ type DBData struct {
 	nodeRentedBy       map[uint64]uint64
 	nodeRentContractID map[uint64]uint64
 
-	nodes              map[uint64]node
-	nodeTotalResources map[uint64]node_resources_total
-	farms              map[uint64]farm
-	twins              map[uint64]twin
-	publicIPs          map[string]public_ip
-	publicConfigs      map[uint64]public_config
-	nodeContracts      map[uint64]node_contract
-	rentContracts      map[uint64]rent_contract
-	nameContracts      map[uint64]name_contract
-	billings           map[uint64][]contract_bill_report
-	contractResources  map[string]contract_resources
-
-	db *sql.DB
+	nodes               map[uint64]node
+	nodeTotalResources  map[uint64]node_resources_total
+	farms               map[uint64]farm
+	twins               map[uint64]twin
+	publicIPs           map[string]public_ip
+	publicConfigs       map[uint64]public_config
+	nodeContracts       map[uint64]node_contract
+	rentContracts       map[uint64]rent_contract
+	nameContracts       map[uint64]name_contract
+	billings            map[uint64][]contract_bill_report
+	contractResources   map[string]contract_resources
+	nonDeletedContracts map[uint64][]uint64
```

Comment:
this could have been just a counter, but i used a map anyway just in case we wanted to use it in the future